keps/sig-storage/1710-selinux-relabeling/README.md

```diff
@@ -0,0 +1,477 @@
+---
+title: Skip SELinux relabeling of volumes
+authors:
+  - "@jsafrane"
+owning-sig: sig-storage
+participating-sigs:
+  - sig-auth
+  - sig-node
+reviewers:
+  - "@msau42"
+  - "@gnufied"
+  - "@rhatdan"
+  - "@haircommander"
+  - "@saschagrunert"
+  - "@liggit"
+  - "@tallclair"
+approvers:
+  - "@saad-ali"
+editor: TBD
+creation-date: 2020-02-18
+last-updated: 2020-05-12
+status: implementable
+see-also:
+  - /keps/sig-storage/20200120-skip-permission-change.md
+replaces:
+superseded-by:
+
+stage: alpha
+latest-milestone: "v1.19"
+milestone:
+  alpha: "v1.19"
+  beta: "v1.20"
+  stable: "v1.22"
+feature-gate:
+  name: SELinuxRelabelPolicy
+  components:
+    - kube-apiserver
+    - kubelet
+rollback-supported: true
+metrics:
+  # TODO: fill at beta
+
+---
+
+# Skip SELinux relabeling of volumes
+
+## Table of Contents
```

Comment:
Can you also add the new production readiness questionaire?

Reply:
Added. Answered all alpha question + some beta ones.